src/plots/attributes.js

```diff
@@ -41,6 +41,19 @@ module.exports = {
             'when toggling legend items.'
         ].join(' ')
     },
+    legendrank: {
+        valType: 'number',
+        dflt: 1000,
+        editType: 'style',
+        description: [
+            'Sets the legend rank for this trace.',
+            'Items and groups with smaller ranks would be presented on top/left side while',
```

Comment:
replace "would be" with "are" please, throughout :)

Reply:
Done in https://github.com/plotly/plotly.js/pull/5591/commits/a1f87b9dbe93a83e3f91da4f3599b22e9a4c166e